servicemesh-install/topics/appendix-crd-old.adoc

```diff
@@ -0,0 +1,283 @@
+[[creating-custom-resource-pre-tp10]]
+= Creating a custom resource file (0.9.TechPreview)
+
+To deploy the {ProductShortName} control plane, you must deploy a custom resource. A _custom resource_ is an object that extends the Kubernetes API, or allows you to introduce your own API into a project or a cluster. You define a custom resource as a yaml file that defines the object, and then you use the yaml file to create the object. The following example contains all of the supported parameters and deploys {ProductName} {ProductVersion} images based on Red Hat Enterprise Linux (RHEL).
+
+[TIP]
+====
+Deploying an *_istio-installation.yaml_* file that includes all of the parameters ensures that you have installed all of the Istio components that are required to complete the tutorials included in this document.
+====
+
+[IMPORTANT]
+====
+The 3scale Istio Adapter is deployed and configured in the custom resource file. It also requires a working 3scale account (https://www.3scale.net/signup/[SaaS] or https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.4/html/infrastructure/onpremises-installation[On-Premises]).
+====
+
+.Full example istio-installation.yaml
+
+[source,yaml]
+----
+apiVersion: "istio.openshift.com/v1alpha1"
+kind: "Installation"
+metadata:
+  name: "istio-installation"
+spec:
+  deployment_type: openshift
+  istio:
+    authentication: true
+    community: false
+    prefix: openshift-istio-tech-preview/
+    version: 0.10.0
+  jaeger:
+    prefix: distributed-tracing-tech-preview/
+    version: 1.11.0
+    elasticsearch_memory: 1Gi
+  kiali:
+    username: username
+    password: password
+    prefix: openshift-istio-tech-preview/
+    version: 0.16.2
```

Comment:
we may need to add a new version of Kiali as well - I'm waiting for confirmation.

Reply:
Confirmed with @brian-avery that 0.16.2 is correct for TP10.

Reply:
0.16.2 is shipping with TP10.